### PR TITLE
Add kong gateway logs to forward logs section

### PR DESCRIPTION
**Pull request by gunnalahema**
This PR contains document for forwarding kong gateway logs to new relic using file log plugin.

### CODE REVIEW

**Review by nr-rkallempudi — APPROVED** [review]

**Review by hrai-nr — APPROVED** [review]

**Review by rohit-bandlamudi-nr — APPROVED** [review]

File: src/content/docs/logs/forward-logs/kong-gateway.mdx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+metaDescription: How to forward your logs to New Relic using our kong Gateway file log plugin, so you can use enhanced log management capabilities.
+
+freshnessValidatedDate: never
+---
+
+Forwarding Kong Gateway logs to New Relic provides valuable insights into API gateway performance, health, security, and usage. This document outlines the process of using the File Log plugin to forward logs from Kong Gateway to New Relic.
+
+## Prerequisites
+
+* Kong Gateway with installed [Custom Resource Definitions (CRDs)] (https://docs.konghq.com/kubernetes-ingress-controller/latest/plugins/rate-limiting/#install-the-gateway-apis)
```
Comment (akristen) [inline src/content/docs/logs/forward-logs/kong-gateway.mdx]: @gunnalahema Are there other compatibility requirements around OS, versioning, and so on?

Comment (gunnalahema) [reply src/content/docs/logs/forward-logs/kong-gateway.mdx]: No

File: src/content/docs/logs/forward-logs/kong-gateway.mdx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+freshnessValidatedDate: never
+---
+
+Forwarding Kong Gateway logs to New Relic provides valuable insights into API gateway performance, health, security, and usage. This document outlines how to direct the File Log plugin from Kong Gateway to forward logs from Kong Gateway to New Relic.
+
+## Compatibility and requirements [#compatibility-requirements]
+
+To use this log forwarding plugin, you need the following:
+
+    * [Kubernetes plugin] (https://docs.newrelic.com/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/) installed
```
Comment (akristen) [inline src/content/docs/logs/forward-logs/kong-gateway.mdx:17]:
```suggestion
    * [Kubernetes plugin] (/docs/logs/forward-logs/kubernetes-plugin-log-forwarding) installed 
```

File: src/content/docs/logs/forward-logs/kong-gateway.mdx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+---
+
+Forwarding Kong Gateway logs to New Relic provides valuable insights into API gateway performance, health, security, and usage. This document outlines how to direct the File Log plugin from Kong Gateway to forward logs from Kong Gateway to New Relic.
+
+## Compatibility and requirements [#compatibility-requirements]
+
+To use this log forwarding plugin, you need the following:
+
+    * [Kubernetes plugin] (https://docs.newrelic.com/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/) installed 
+    * Kong Gateway installed with [Custom Resource Definitions (CRDs)] (https://docs.konghq.com/kubernetes-ingress-controller/latest/plugins/rate-limiting/#install-the-gateway-apis)
```
Comment (akristen) [inline src/content/docs/logs/forward-logs/kong-gateway.mdx:18]:
```suggestion
    * Kong Gateway installed with [Custom Resource Definitions (CRDs)](https://docs.konghq.com/kubernetes-ingress-controller/latest/plugins/rate-limiting/#install-the-gateway-apis)
```

File: src/content/docs/logs/forward-logs/kong-gateway.mdx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    Deploy the File Log plugin configuration to your Kubenrnetes cluster, but be sure to update `file-log-plugin.yaml` with the actual filename of your manifest:
+
+        ```bash
+            kubectl apply -f file-log-plugin.yaml -n kong
+        ```
+    </Step>
+    <Step>
+    ## Confirm that your logs are in the New Relic platform [#confirm-logs-platform]
+
+    Once you've deployed the configurations, go to **[one.newrelic.com > All Capabilities > Logs](https://one.newrelic.com/launcher/logger.log-launcher)**. Confirm that logs from Kong Gateway are appearing in the platform by running this query:
```
Comment (akristen) [inline src/content/docs/logs/forward-logs/kong-gateway.mdx:64]:
```suggestion
    Once you've deployed the configurations, go to **[one.newrelic.com > All Capabilities > Logs](https://one.newrelic.com/launcher/logger.log-launcher)**. Confirm that logs from Kong Gateway are appearing in the platform by triggering some data, then running a query that filters for that data.
    
 You can start building that query with this example:
```

**Review by akristen — APPROVED** [review]

**Pull request merged**